tests/structured_conf/test_structured_config.py

```diff
@@ -535,6 +537,52 @@ def test_any(name: str) -> None:
             Color.BLUE,
         ]
 
+    def test_tuple_examples(self, class_type: str) -> None:
```

Comment:
This new test could be merged with test_list_examples. Do you prefer it this way so its separated?

Reply:
If it's a near duplicate try to merge them. (it should be because underneeth Tuple and List are both converted to the same ListConfig.

Reply:
@omry done